get_gcs_test.go

```diff
@@ -8,35 +8,17 @@ import (
 	"testing"
 )
 
-// initGCPCredentials writes a temporary GCS credentials file if necessary and
```

Comment:
Similar to my other comment for S3, we should documentation the expectation for where/how these tests should run. It won't be obvious if someone tries to run the tests locally for this module.